solutions/cses-1745.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: cses-1745
+source: CSES
+title: Money Sums
+author: Sofia Yang
+---
+
+[Codeforces Editorial](https://codeforces.com/blog/entry/70018)
+
+<LanguageSection>
+
+<JavaSection>
+
+## Solution
+
+**Time Complexity:** $\mathcal{O}(N\cdot X)$
+
+## Java Implementation
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class MoneySums {
+	static class Kattio extends PrintWriter {
```

Comment:
this Kattio is different, but using the codesnip kattio should still work without any modifications to your code, right? if the [codesnip kattio](https://usaco.guide/general/working-mdx?lang=java#kattio) works, could you replace this with `// CodeSnip{Kattio}` so it can be snipped out?

Reply:
https://github.com/cpinitiative/usaco-guide/pull/1640#discussion_r642700084

Reply:
no i think she replaced it with the same kattio now; so it can be snipped out?